pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/ClusterTest.java

```diff
@@ -259,7 +259,9 @@ protected void startServers(int numServers)
       throws Exception {
     FileUtils.deleteQuietly(new File(TEMP_SERVER_DIR));
     for (int i = 0; i < numServers; i++) {
-      _serverStarters.add(startOneServer(i));
+      BaseServerStarter serverStarter = startOneServer(i);
+      _serverStarters.add(serverStarter);
+      _helixAdmin.enableInstance(getHelixClusterName(),serverStarter.getInstanceId(), true);
```

Comment:
This is needed, otherwise integration test 
`ControllerPeriodicTasksIntegrationTest.testSegmentStatusChecker` will fail